packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -92,66 +96,69 @@ Cons:
 
 ### API
 
-#### Icon column element:
+#### Enum column element:
 
 _Component Name_
 
--   `nimble-table-column-icon`
+-   `nimble-table-column-enum`
 
 _Props/Attrs_
 
 -   `field-name`: string
 -   `key-type`: 'string' | 'number' | 'boolean'
+-   `fixed-to-icon-width`: boolean - When set, the column will have a fixed width that makes the column the appropriate width to render only a single icon in the cell. This should only be set when the header contains a single icon (no text) and none of the child mapping elements will result in text being rendered in a cell.
 
 _Content_
 
--   column title (icon)
--   1 or more `nimble-mapping-icon` or `nimble-mapping-spinner` elements
-
-#### General mapping column element:
-
-_Component Name_
-
--   `nimble-table-column-enum-text`
-
-_Props/Attrs_
-
--   `field-name`: string
--   `key-type`: 'string' | 'number' | 'boolean'
--   `fractional-width`: number (defaults to 1)
--   `min-pixel-width`: number (defaults to minimum supported by table)
-
-_Content_
-
--   column title (text or icon)
--   1 or more `nimble-mapping-text` elements
+-   column title (icon and/or text)
+-   1 or more `nimble-mapping-icon`, `nimble-mapping-spinner`, `nimble-mapping-text`, or `nimble-mapping-placeholder` elements
 
 #### Mapping element (icon):
 
+The icon mapping element will support displaying an icon, icon with text, or only text in a cell. If only text is displayed in a cell, space will be reserved for an icon so that the text associated with all icon mapping elements is aligned. A group row associated with an icon mapping element will always display the icon, if specified, and text.
+
 _Component Name_
 
 -   `nimble-mapping-icon`
 
 _Props/Attrs_
 
 -   `key`: string | number | boolean | undefined
--   `icon`: string - name of the Nimble icon element
+-   `icon`: string | undefined - name of the Nimble icon element. If `undefined`, no icon will be associated with the given `key`, but space will be reserved in the cell for an icon.
```

Comment:
If a client doesn't want empty space reserved for missing icons, they can just use `nimble-mapping-text` instead, right?

Reply:
Yes, that is correct.